src/win32/InstallerScripts.vbs

```diff
@@ -422,3 +422,32 @@ Private Function GetUserSID()
         GetUserSID = oWMI.SID
     End If
 End Function
+
+Public Function VerifyOrCreateRegistryKey()
+    On Error Resume Next
+    Dim strKeyPath, oReg
+    Dim objCtx, objLocator, objServices
+    Const HKEY_LOCAL_MACHINE = &H80000002
+
+    Set objCtx = CreateObject("WbemScripting.SWbemNamedValueSet")
+    objCtx.Add "__ProviderArchitecture", 64
+    objCtx.Add "__RequiredArchitecture", True
+
+    Set objLocator = CreateObject("WbemScripting.SWbemLocator")
+    Set objServices = objLocator.ConnectServer(".", "root\default", "", "", , , , objCtx)
+    Set oReg = objServices.Get("StdRegProv")
+
+    strKeyPath = "SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\wazuh-agent.exe"
```

Comment:
Why is it not being used the whole registry path?

Reply:
`HKEY_LOCAL_MACHINE` is set when the variable `Const HKEY_LOCAL_MACHINE = &H80000002` is defined, this ensures that the Key is created in the `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\wazuh-agent.exe` registry